src/components/portal-target.js

```diff
@@ -14,13 +17,13 @@ export default {
   },
   data () {
     return {
-      transports,
+      transports: wormhole.transports,
       firstRender: true,
     }
   },
   mounted () {
     if (!this.transports[this.name]) {
-      this.$set(this.transports, this.name, undefined)
+      this.$set(this.transports, this.name, [])
```

Comment:
Unrelated to the feature, but I think this whole `if` block (but not the rest) should be in `created()` rather than `mounted` so it's applied before first render.